test/end-to-end/browser/helpers/complete-risk-assessment.js

```diff
@@ -207,3 +216,37 @@ export const thenTheAssessmentIsCompleted = (config = defaultAssessmentConfig) =
     });
   }
 };
+
+
+export const andICanViewTheAssessmentAgain = (config = defaultAssessmentConfig) => {
```

Comment:
Is there any benefit in pulling this function out into its own file?

Reply:
not much no